mono/mini/mini-windows.c

```diff
@@ -301,7 +500,11 @@ thread_timer_expired (HANDLE thread)
 		guchar *ip;
 
 #ifdef _WIN64
-		ip = (guchar *) context.Rip;
+#ifdef TARGET_ARM64
+		ip = (guchar*)context.Pc;
+#else //TARGET_AMD64
```

Comment:
Nit, can you do positive check here for `TARGET_AMD64` and error in the else case?